ynput/OpenPype
# Max : Optional validator to check invalid context data

## Changelog Description
Add optional validator check on invalid context data for asset and task in 3dsMax

## Additional info
Reference to maya's `Validate Instance in Context`

## Testing notes:
1. Launch 3dsMax via launcher
2. Create two instances with same families, one of them set different asset and task
3. Publish
4. Error would be raised if you enable the validator which check on invalid context data
5. Repair it 
6. Run publisher again it will pass without error
